google/resource_compute_ssl_policy.go

```diff
@@ -0,0 +1,226 @@
+package google
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	computeBeta "google.golang.org/api/compute/v0.beta"
+)
+
+func resourceComputeSslPolicy() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceComputeSslPolicyCreate,
+		Read:   resourceComputeSslPolicyRead,
+		Update: resourceComputeSslPolicyUpdate,
+		Delete: resourceComputeSslPolicyDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(2 * time.Minute),
+			Update: schema.DefaultTimeout(2 * time.Minute),
+			Delete: schema.DefaultTimeout(2 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"custom_features": &schema.Schema{
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				Set:      schema.HashString,
+			},
+
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true, // currently, the beta patch API call does not allow updating the description
+			},
+
+			"min_tls_version": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "TLS_1_0",
+				// Although compute-gen.go says that TLS_1_3 is a valid value, the API currently (26 Mar 2018)
+				// responds with an HTTP 200 but doesn't actually create/update the policy.
```

Comment:
Oh that's distressing. Is there an issue in the public issue tracker that we could link to here?

Reply:
Not that I'm aware of, but I can file one if you'd like! Relevant lines are 21109-21118 in compute/v0.beta/compute-gen.go.

Reply:
If you don't mind, that would be great! That way if someone stumbles across this part of the code and sees a comment with a link to an issue that's been fixed, we'd be able to add TLS_1_3 real quick

Reply:
Ok, can do! Which product should I file it under? Cloud SDK?

Reply:
I'm not super familiar with how the products are laid out in components, but if there's one for compute admin api, or networking api, or something along those lines. I'm not going to block this PR on getting that link in there though. I'll finish up the review now :)

Reply:
No worries, I just added a link to the issue that Aaron created. (Thanks Aaron!)